lib/active_merchant/billing/gateways/bpoint.rb

```diff
@@ -61,10 +61,10 @@ def refund(amount, authorization, options={})
         commit(request_body)
       end
 
-      def void(amount, authorization, options={})
+      def void(authorization, options={})
```

Comment:
Do we need to update any code that was previously calling `void` with `amount` as the first argument?

Reply:
So the entire reason for this bug is that Core never includes `amount` when making a `void` request; it just sends the authorization and the options hash. I have a separate PR out for the Core half of this problem, which gets around this issue by passing the required `amount` field within the `options` hash. 

I'm not aware of anywhere else that this method is called.

Reply:
Yep, I checked out that other PR, and kinda realized this, too. Interesting that voids never worked for BPoint 🤔

Reply:
Sadly, BPoint is not alone in this... In investigating this, Chinh and I realized that Bambora Apac and Merchant Warrior have the same problem for their `void` transactions. We created new bug tickets to resolve those while we're at it. 

This problem seems to stem from the fact that we don't have any automated end-to-end testing that runs through both Core and Active Merchant.

Reply:
Setting up that kind of automated integration tests would be a big effort, but I wonder if the payoff would be worth it. I _kinda_ think it would.